tests/cli/open_package/test_export.py

```diff
@@ -0,0 +1,172 @@
+Open package '{pkg_path}'...
+Export package to '{export_path}'...
+Finished with errors!
+""")
+    assert not os.path.exists(cli.abspath(export_path))
+
+
+@pytest.mark.parametrize(
+    "pkg_uuid",
+    [
+        "0eaf289c-166d-4bd9-a4ba-dbf6bbc76ef1",
+    ],
+)
+def test_export_package_to_png_relative(cli, pkg_uuid):
+    """
+    Test exporting a package to PNG format with a relative path.
+    """
+    library = params.POPULATED_LIBRARY
+    cli.add_library(library.dir)
+    pkg_path = os.path.join(library.dir, "pkg", pkg_uuid)
+
+    # Use substitution patterns to avoid conflicts
+    export_path = os.path.join("footprints", "{{FOOTPRINT}}.png")
+    code, stdout, stderr = cli.run("open-package", "--export", export_path, pkg_path)
+
+    assert code == 0
+    assert stderr == ""
+
+    # Check that all footprints were exported
+    footprint_dir = cli.abspath("footprints")
+    assert os.path.isdir(footprint_dir)
+    exported_files = os.listdir(footprint_dir)
+    # This package has 2 footprints, so we expect 2 PNGs
+    assert len(exported_files) == 2
+    for f in exported_files:
+        assert f.endswith(".png")
+        assert os.path.getsize(os.path.join(footprint_dir, f)) > 0
+
+
+@pytest.mark.parametrize(
+    "pkg_uuid",
+    [
+        "0eaf289c-166d-4bd9-a4ba-dbf6bbc76ef1",
+    ],
+)
+def test_export_package_to_svg_absolute(cli, pkg_uuid):
+    """
+    Test exporting a package to SVG format with an absolute path.
+    """
+    library = params.POPULATED_LIBRARY
+    cli.add_library(library.dir)
+    pkg_path = os.path.join(library.dir, "pkg", pkg_uuid)
+
+    # Use substitution patterns to avoid conflicts
+    export_dir = cli.abspath("svg_exports")
+    export_path = os.path.join(export_dir, "{{FOOTPRINT}} with spaces.svg")
+    code, stdout, stderr = cli.run("open-package", "--export", export_path, pkg_path)
+
+    assert code == 0
+    assert stderr == ""
+
+    # Check that all footprints were exported
+    assert os.path.isdir(export_dir)
+    exported_files = os.listdir(export_dir)
+    assert len(exported_files) == 2
+    for f in exported_files:
+        assert f.endswith(".svg")
+        assert " with spaces" in f
+        assert os.path.getsize(os.path.join(export_dir, f)) > 0
+
+
+def test_if_output_directories_are_created(cli):
+    """
+    Test if output directories are created automatically.
+    """
+    library = params.POPULATED_LIBRARY
+    cli.add_library(library.dir)
+    pkg_uuid = "0eaf289c-166d-4bd9-a4ba-dbf6bbc76ef1"
+    pkg_path = os.path.join(library.dir, "pkg", pkg_uuid)
+
+    export_dir = cli.abspath("nonexistent/nested/dir")
+    export_path = os.path.join(export_dir, "{{FOOTPRINT}}.pdf")
+    assert not os.path.exists(export_dir)
+
+    code, stdout, stderr = cli.run("open-package", "--export", export_path, pkg_path)
+
+    assert code == 0
+    assert stderr == ""
+    assert os.path.isdir(export_dir)
+
+    exported_files = os.listdir(export_dir)
+    assert len(exported_files) == 2
```

Comment:
Same here, don't check the *number* of generated files, check their *names* as a `set`.